hw2/03-form.js

```diff
@@ -1,3 +1,76 @@
 /** Exercise 03 - Form **/
 
-// Add your code here
+const validateForm = (name, email) => {
```

Comment:
Code is clean and modular. Not entering an email or name pops up with an error message. Your tooltip for the email formatting is fantastic. I didn't even know you could do that and I want to go add it into my code now haha. Output to console is correct. May want to consider using `console.group()` for output to console, but I'm not sure if it's required or not. Very good code in my opinion!